app/src/main/java/com/g11x/checklistapp/NotificationListActivity.java

```diff
@@ -1,19 +1,116 @@
 package com.g11x.checklistapp;
 
 import android.os.Bundle;
-import android.support.design.widget.FloatingActionButton;
-import android.support.design.widget.Snackbar;
 import android.support.v7.app.AppCompatActivity;
-import android.support.v7.widget.Toolbar;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.helper.ItemTouchHelper;
+import android.view.LayoutInflater;
 import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
 
-//import com.g11x.checklistapp.rst.R;
+import com.g11x.checklistapp.data.Notification;
 
-public class NotificationListActivity extends AppCompatActivity {
+public class NotificationListActivity extends NavigationActivity {
+  private RecyclerView mRecyclerView;
+  private RecyclerView.Adapter mAdapter;
+  private RecyclerView.LayoutManager mLayoutManager;
 
+  @Override
+  protected int getNavDrawerItemIndex() {
+    return NAVDRAWER_ITEM_NOTIFICATIONS;
+  }
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_notification_list);
+    mRecyclerView = (RecyclerView) findViewById(R.id.my_recycler_view);
+
+    // use this setting to improve performance if you know that changes
+    // in content do not change the layout size of the RecyclerView
+    mRecyclerView.setHasFixedSize(true);
+
+    // use a linear layout manager
+    mLayoutManager = new LinearLayoutManager(this);
+    mRecyclerView.setLayoutManager(mLayoutManager);
+
+    ItemTouchHelper helper = new ItemTouchHelper(new ItemTouchHelper.SimpleCallback(0, ItemTouchHelper.RIGHT) {
```

Comment:
Hmm, swiping left is often used for a different action than right. I've been pretty trained on this with Wear. Were you thinking LEFT would do something other than dismiss?